cloud-storage/onedrive/api/src/java/org/sakaiproject/onedrive/service/OneDriveService.java

```diff
@@ -0,0 +1,55 @@
+package org.sakaiproject.onedrive.service;
+
+import java.util.List;
+
+import org.sakaiproject.onedrive.model.OneDriveItem;
+import org.sakaiproject.onedrive.model.OneDriveUser;
+
+/**
+ * Interface for communicating with the OneDrive API.
+ */
+public interface OneDriveService {
+
+	// ONEDRIVE CONSTANTS
+	public final String ONEDRIVE_PREFIX = "onedrive.";
```

Comment:
Constants should be moved to the implementation of the service if they should be public meaning that other area's of Sakai would need to use these constants outside the service then lets put them in their own CouldStorageConstants class. Attaching these to the service interface is usually not ideal.

Reply:
Can't do that, they are used in some places outside the impl.